bin/yb-ctl

```
@@ -1022,48 +1070,135 @@ class ClusterControl:
                     raise subprocess.CalledProcessError(proc.returncode,
                                                         cmd_list_tservers, output=output)
                 num_yb_admin_ts = len(output.splitlines()) - 1
-                # This will not work if you have stopped/removed a node and the master is still
-                # aware of it because we do not have a yb-admin API to return only live tablet
-                # servers.
                 if num_yb_admin_ts == num_alive_ts:
+                    # This will not work if you have stopped/removed a node and the master is still
+                    # aware of it because we do not have a yb-admin API to return only live tablet
+                    # servers.
                     return True
+                elif num_yb_admin_ts < 0:
+                    # We might get no output from yb-admin if the leader is not up yet...
+                    logging.info("Master leader election still pending...")
+                else:
+                    logging.info("Waiting for all tablet servers to register: {}/{}".format(
+                        num_yb_admin_ts, num_alive_ts))
             except subprocess.CalledProcessError:
```

Ditto regarding suppressing errors.